dumb-jump.el

```diff
@@ -1168,21 +1166,31 @@ If `nil` always show list of more than 1 match."
   "Use TEST as the standard input for the CMD."
   (with-temp-buffer
     (insert test)
-    (shell-command-on-region (point-min) (point-max) cmd nil t)
+    (shell-command-on-region (point-min) (point-max) cmd (current-buffer) t)
```

Comment:
Is it necessary to use `(current-buffer)`? I can see in the doc on `shell-command-on-region` that:
> If the value is nil, use the buffer ‘*Shell Command Output*’.

Reply:
Ok, right!